vendor/github.com/nmstate/kubernetes-nmstate/api/shared/nodenetworkconfigurationpolicy_types.go

```diff
@@ -42,13 +42,15 @@ type NodeNetworkConfigurationPolicyStatus struct {
 }
 
 const (
-	NodeNetworkConfigurationPolicyConditionAvailable ConditionType = "Available"
-	NodeNetworkConfigurationPolicyConditionDegraded  ConditionType = "Degraded"
+	NodeNetworkConfigurationPolicyConditionAvailable   ConditionType = "Available"
```

Comment:
@qinqon  seems odd to change vendor. Is it okay or should we in case of updating the API first update the API, merge the update and when the update is vendored, use it in code?

Reply:
It's not possible to do on the same PR at a pair of commits ?

Reply:
Ah I see, thanks. I've splitted it to separate commits